scripts/portconfig

```diff
@@ -126,6 +126,13 @@ class portconfig(object):
     def set_fec(self, port, fec):
         if self.verbose:
             print("Setting fec %s on port %s" % (fec, port))
+        supported_fecs = self.get_supported_fecs(port)
+        if fec not in supported_fecs:
```

Comment:
If the vendor SAI doesn't support getting supported FEC modes, does it mean they cannot configure FEC? I believe the logic should be backward compatible. If supported_fec modes exists in STATE_DB we should use that to validate FEC. Else we should fall back to existing logic.

Reply:
Hi @dgsudharsan 
It is backward compatible.
In case fetching supported FEC mode is not supported by the vendor, no `supported_fec` is exposed to STATE_DB. In this case, no checking is performed and the FEC can be configured.